source/index.html.md

```diff
@@ -1,239 +1,674 @@
+}
 ```
 
-```shell
-curl "http://example.com/api/kittens/2"
-  -X DELETE
-  -H "Authorization: meowmeowmeow"
+### IN
+
+<div class="center-column"></div>
+```sql
+{
+  SELECT *
+  FROM data-set-id/en-0
+  WHERE country IN ('Mexico', 'Canada', ...)
+}
 ```
 
-```javascript
-const kittn = require('kittn');
+### BETWEEN
 
-let api = kittn.authorize('meowmeowmeow');
-let max = api.kittens.delete(2);
+<div class="center-column"></div>
+```sql
+{
+  SELECT *
+  FROM data-set-id/en-0
+  WHERE liquidation_date BETWEEN 2016-01-01 AND 2018-01-01
+}
 ```
 
-> The above command returns JSON structured like this:
+## GROUP BY and HAVING
+
+<div class="center-column"></div>
+```sql
+{
+  SELECT COUNT(customer_id), country
+  FROM data-set-id/en-0
+  GROUP BY country
+  HAVING COUNT(customer_id) > 100 
+}
+```
+
+### Subselects
+
+<div class="center-column"></div>
+```sql
+{
+  SELECT *
+  FROM data-set-id/en-0
+  WHERE total_count = [ANY|ALL] (SELECT COUNT(customer_id) FROM data-set-id2/en-1)
+}
+```
 
-```json
+<div class="center-column"></div>
+```sql
 {
-  "id": 2,
-  "deleted" : ":("
+  SELECT *
+  FROM (SELECT customer_id, parent_account_id, purchase_total FROM data-set-id2/en-1)
+  AS subSelect
+  WHERE purchase_total > 1500
 }
 ```
+  
+## Geospatial Features
+
+Geometry properties for data sets are stored as `geojson`, and will be rendered as that unless specified. You can do this using the transformation functions `ST_GeomFromText` to create geometry objects, which can then be manipulated and transformed. Use `ST_AsGeoJSON` or `ST_AsText` in order to turn the final result back to text from binary.  
+
+Example (where `geometry_property` is a property from the data set of type geometry. This information can be obtained in the API Info tab when viewing a data set):
+
+<code>SELECT ST_AsGeoJSON(ST_GeomFromText(geometry_property))<br/>FROM data-set-id/en-0</code>
+
+### Supported Functions
+
+>`ST_AsGeoJSON`<br/>
+>`ST_AsJson`<br/>
+>`ST_AsText`<br/>
+>`ST_Buffer`<br/>
+>`ST_Contains`<br/>
+>`ST_Crosses`<br/>
+>`ST_Difference`<br/>
+>`ST_Disjoint`<br/>
+>`ST_Distance`<br/>
+>`ST_DWithin`<br/>
+>`ST_Envelope`<br/>
+>`ST_Equals`<br/>
+>`ST_GeomFromText`<br/>
+>`ST_GeomFromTextSrid`<br/>
+>`ST_Intersects`<br/>
+>`ST_Overlaps`<br/>
+>`ST_PointFunc`<br/>
+>`ST_Relate`<br/>
+>`ST_Touches`<br/>
+>`ST_Transform`<br/>
+>`ST_Union`<br/>
+>`ST_UnionAggregate`<br/>
+>`ST_Within`<br/>
+>`ST_XFunc`<br/>
+>`ST_XMax`<br/>
+>`ST_XMin`<br/>
+>`ST_YFunc`<br/>
+>`ST_YMax`<br/>
+>`ST_YMin`<br/>
+
+We are very interested in expanding the geospatial capabilities of **NiQL**. If there is additional functionalities, or issues with the the implementations, please do not hesitate to <a href="https://namara.io/contact" target="_blank" rel="noreferrer noopener">reach out to us</a>.
+
+<aside class="notice">Please refer to the <a href="https://postgis.net/docs/" target="_blank" rel="noreferrer noopener">PostGIS documentation</a> for the functional specifics</aside>
```

Comment:
My bad, we should probably be dropping them on this page `https://postgis.net/docs/reference.html`

Reply:
done